pkg/schema/field.go

```diff
@@ -0,0 +1,12 @@
+package schema
+
+type FieldType byte
+
+const (
+	FieldTypeBinary = 1
```

Comment:
let's follow the bson type definition. https://github.com/casbin-mesh/neo/blob/feat/btuple-weny/pkg/primitive/bsontype/bsontype.go